cypress/support/pages/article.pageObject.js

```diff
@@ -0,0 +1,46 @@
+import PageObject from '../PageObject';
+
+class ArticlePageObject extends PageObject {
+  url = 'article';
+
+  get titleOfArticle() {
+    return cy.get('h1');
+  }
+
+  get editButton() {
+    return cy.get('.container > .article-meta > ' +
+    ':nth-child(3) > .btn-outline-secondary');
+  }
+
+  get deleteButton() {
+    return cy.get('.container > .article-meta > ' +
+    ':nth-child(3) > .btn-outline-danger');
```

Comment:
Similarly, the CSS selector for the `deleteButton` is very specific. If possible, use a more robust selector, such as a data attribute, to ensure the test remains stable even if the HTML structure changes.